cli/commands/benchmark-web-vitals.mjs

```diff
@@ -161,13 +161,28 @@ export async function handler( opt ) {
 
 	const browser = await puppeteer.launch( { headless: 'new' } );
 
+	// Log progress only under certain conditions (multiple URLs to benchmark).
+	const logProgress = logURLProgress( opt );
+
 	for await ( const url of getURLs( opt ) ) {
-		const { completeRequests, metrics } = await benchmarkURL(
-			browser,
-			params
-		);
+		if ( logProgress ) {
+			log( `Benchmarking URL ${ url }...` );
```

Comment:
Using `process.stderr.write` won't automatically add a newline, so if doing this:

```suggestion
			process.stderr.write( `Benchmarking URL ${ url }...` );
```

Then down below the success/error can be displayed on the same line.

Reply:
It looks like this would also be the case for `process.stdout.write`, this is related to the other comments above. But I definitely like the idea of keeping this in one line.

I'm not sure what is the tradeoffs between using `console.*` functions vs `process.stdout/stderr` in this context. We should probably discuss that and then decide how to proceed by updating the `log` and `formats` functions accordingly.